gapic/schema/wrappers.py

```diff
@@ -821,13 +822,64 @@ def field_headers(self) -> Sequence[str]:
 
         return next((tuple(pattern.findall(verb)) for verb in potential_verbs if verb), ())
 
+    def http_rule_to_tuple(self, http_rule: http_pb2.HttpRule) -> Tuple[str, str, str]:
```

Comment:
Minor digression: I think it would be more helpful to return either a dataclass or a named tuple. I'd prefer to make method, uri, body named fields instead of positional convention.

Reply:
Done - rewrote as dataclass.